ReactWindows/ReactNative.Shared/UIManager/BorderedViewParentManager.cs

```diff
@@ -65,34 +68,46 @@ public void SetBorderRadius(Border view, int index, double radius)
                     break;
             }
 
+#pragma warning disable WPF0041
             view.CornerRadius = cornerRadius;
+#pragma warning restore WPF0041
```

Comment:
This is not true, all DependencyObjects in WPF have the SetCurrentValue method, you're probably targeting UWP when discovering this.  You have a couple options here, you can #if in the WPF behavior, or you can make all view managers separate so you don't have to #if anything.  I've been advocating for this for a while because UWP and WPF are different frameworks that, yes, do share some syntax, but will more often than not need to diverge in behavior.